pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -338,6 +356,71 @@ func (f *framework) RunScorePlugins(pc *PluginContext, pod *v1.Pod, nodes []*v1.
 	return pluginToNodeScoreMap, nil
 }
 
+// RunNormalizeScorePlugins runs the normalize score plugins. It should be called after
+// RunScorePlugins with the PluginToNodeScoreMap result. It then modifies the map with
+// normalized scores. It returns a non-success Status if any of the normalize score plugins
+// returns a non-success status.
+func (f *framework) RunNormalizeScorePlugins(pc *PluginContext, pod *v1.Pod, scores PluginToNodeScoreMap) *Status {
+	ctx, cancel := context.WithCancel(context.Background())
+	errCh := schedutil.NewErrorChannel()
+	workqueue.ParallelizeUntil(ctx, 16, len(f.normalizeScorePlugins), func(index int) {
+		pl := f.normalizeScorePlugins[index]
+		nodeScoreList, ok := scores[pl.Name()]
+		// During framework initialization, we make sure that NormalizeScore plugins are
+		// a subset of Score plugins. So nodeScoreList should exist given that implementation
+		// of RunScorePlugins produce the correct output.
+		if !ok {
+			err := fmt.Errorf("NormalizeScore plugin %v has no corresponding scores in the PluginToNodeScoreMap.", pl.Name())
```

Comment:
nit: s/NormalizeScore/normalize score/
nit: remove the `.` in the end

errors should start with lower case letter